assets/js/phoenix_live_view/dom.js

```diff
@@ -285,13 +285,16 @@ let DOM = {
     }
   },
 
-  maybeHideFeedback(container, input, phxFeedbackFor){
-    if(!(this.private(input, PHX_HAS_FOCUSED) || this.private(input, PHX_HAS_SUBMITTED))){
-      let feedbacks = [input.name]
-      if(input.name.endsWith("[]")){ feedbacks.push(input.name.slice(0, -2)) }
-      let selector = feedbacks.map(f => `[${phxFeedbackFor}="${f}"]`).join(", ")
-      DOM.all(container, selector, el => el.classList.add(PHX_NO_FEEDBACK_CLASS))
-    }
+  maybeHideFeedback(container, inputs, phxFeedbackFor){
+    let selector;
+    inputs.forEach(input => {
+      if(!(this.private(input, PHX_HAS_FOCUSED) || this.private(input, PHX_HAS_SUBMITTED))){
+        let feedbacks = [input.name]
+        if(input.name.endsWith("[]")){ feedbacks.push(input.name.slice(0, -2)) }
+        selector = feedbacks.map(f => `[${phxFeedbackFor}="${f}"]`).join(", ")
```

Comment:
this discards the previous selector. Instead you can accumulate the feedbacks, then join the into a selector on the outside of the forEach

Reply:
Thanks, I've pushed a commit that leverages the feedback for accumulation. I still see improvements on pages containing thousands of inputs; I need a better understanding of LiveView's JS and Morphdom to determine if there is a better way to improve the initial page render for pages that stream thousands of inputs.